docker/server/Dockerfile

```diff
@@ -39,9 +39,11 @@ RUN mkdir /docker-entrypoint-initdb.d
 
 COPY docker_related_config.xml /etc/clickhouse-server/config.d/
 COPY entrypoint.sh /entrypoint.sh
+COPY init-defaults.sh /docker-entrypoint-initdb.d/init-defaults.sh
```

Comment:
By putting that file into that docker-entrypoint-initdb.d folder you will make executing it twice, see 
https://github.com/ClickHouse/ClickHouse/blob/615104c8202adc2b5b15977b65d426ce0b9f57ed/docker/server/entrypoint.sh#L84

Reply:
My bad, fixed